ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/DevMachineHostNameMacro.java

```diff
@@ -60,7 +60,7 @@ public String getDescription() {
     Optional<MachineImpl> devMachine = wsAgentServerUtil.getWsAgentServerMachine();
 
     if (devMachine.isPresent()) {
-      String hostName = devMachine.get().getProperties().get("config.hostname");
```

Comment:
This code can't be used anymore- we don't have such a property anymore. @vparfonov Please take a look and remove it on an occasion.